test/unit/org/apache/cassandra/db/compaction/NeverPurgeTest.java

```diff
@@ -65,7 +65,7 @@ public void neverPurgePartitionTombstoneTest() throws Throwable
     @Test
     public void minorNeverPurgeTombstonesTest() throws Throwable
     {
-        createTable("CREATE TABLE %s (a int, b int, c text, PRIMARY KEY (a, b)) WITH gc_grace_seconds = 0");
+        createTable("CREATE TABLE %s (a int, b int, c text, PRIMARY KEY (a, b)) WITH gc_grace_seconds = 0 AND COMPACTION={'class':'SizeTieredCompactionStrategy'}");
```

Comment:
IIUC UCS doesn't have the same property that "minor never purges tombstones"
perhaps change the test name if the tested property is only a property of STCS? WDYT?